server/src/main/scala/org/http4s/server/middleware/RequestLogger.scala

```diff
@@ -19,16 +19,18 @@ object RequestLogger {
   def apply[F[_]: Effect](
       logHeaders: Boolean,
       logBody: Boolean,
-      redactHeadersWhen: CaseInsensitiveString => Boolean = Headers.SensitiveHeaders.contains
-  )(@deprecatedName('service) routes: HttpRoutes[F])(
-      implicit ec: ExecutionContext = ExecutionContext.global): HttpRoutes[F] =
+      redactHeadersWhen: CaseInsensitiveString => Boolean = Headers.SensitiveHeaders.contains,
+      logAction: String => Unit = logger.info(_)
+  )(@deprecatedName('service) http: Kleisli[F, Request[F], Response[F]])(
+      implicit ec: ExecutionContext = ExecutionContext.global,
```

Comment:
can we remove the default arg implicit here? I don't like this.

Reply:
I missed that, and I agree.